sources/packages/web/src/components/common/applicationTracker/Completed.vue

```diff
@@ -17,6 +17,15 @@
     background-color="warning-bg"
     content="StudentAid BC is reviewing your requested change. If your requested change is approved, your application will be re-evaluated with a new assessment below."
   />
+  <!-- Student request an application change (after COE) - change in progress -->
+  <application-in-progress-change-request
+    icon="fa:fas fa-exclamation-triangle"
```

Comment:
the properties `icon`, `icon-color` and `background-color` doesn't seem to be used by the component. 

![image](https://github.com/user-attachments/assets/c7d0e8b3-4876-494d-bf32-cb88468b2bf4)

Reply:
Properties removed.